charts/base-cluster/templates/backup/velero.yaml

```diff
@@ -77,6 +77,17 @@ spec:
             name: {{ get $existingSecret "name" | required (printf "You need to provide the existingSecret.name for %s" $name) }}
             key: {{ get $existingSecret "key" | default (printf "%s-%s" $providerName $name) }}
           {{- end }}
+          {{- if eq $providerName "aws" }}
+          {{- $providerSpec := get $spec.provider $providerName }}
+          config:
+            {{- if $providerSpec.region }}
+            region: {{ $providerSpec.region | quote }}
+            {{- else }}
+            region: Region1
+            {{- end }}
+            s3ForcePathStyle: {{ ($providerSpec.forcePathStyle | default "true") | quote }}
+            s3Url: {{ $providerSpec.url | quote }}
```

Comment:
Those options are explicitly *not* for aws but for something like minio as aws does not have different URLs as is also referenced in the [velero config](https://velero.io/docs/v1.1.0/api-types/backupstoragelocation/#aws)

You could, and should, rather just adjust the minio provider to have these options, including `region` I guess

Reply:
> I explicitly didn't touch the `minio` configuration to keep backwards compatibility. But I can, off course, add this to `minio` as well. Although I got to admit, that I find the name `minio` a bit misleading, as this configuration is not specific to minio but to most S3 backends (even AWS).

Yeah, we could also rename `minio` to something else if you got something, maybe `genericS3`? 🤔

But the `s3url` is not valid for real aws which is why I want them to be seperate